src/bindings/js/node/include/async_reader.hpp

```diff
@@ -14,11 +14,13 @@ class ReaderWorker : public Napi::AsyncWorker {
      * @brief Constructs ReaderWorker class that is responisible for reading the model asynchronously.
```

Comment:
```suggestion
     * @brief Constructs ReaderWorker class that is responisible for reading the model asynchronously.
     * @note In the Execute() method, the Core object might be used concurrently to call read_model().
```
I think it will be good to add a note for developers here or next to `Execute()` method